src/components/Header/header.styled.tsx

```diff
@@ -0,0 +1,360 @@
+import { styled } from "@mui/system";
+
+export const StyledHeader = styled("header")({
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  backgroundColor: "#FFFFFF",
+  width: "100%",
+  height: 72,
+  fontFamily: "sans-serif",
+  fontWeight: 400,
+  lineHeight: 1.4,
+  fontSize: "16px",
+  color: "#1c1d1f",
+  margin: 0,
+  padding: 0,
+  boxShadow: "0 2px 4px rgba(0,0,0,.08),0 4px 12px rgba(0,0,0,.08)",
+  zIndex: 1010,
+  position: "relative",
+
+  "@media(maxWidth: 480px)": {
+    width: "80%",
+    backgroundColor: "red",
+  },
+});
+
+export const ContentHeader = styled("div")({
+  color: "#1c1d1f",
+  margin: 0,
+  width: "100%",
+  fontWeight: 400,
+  lineHeight: 1.4,
+  fontSize: "14px",
+  background: "#fff",
+  height: "72px",
+  padding: "0 24px",
+  position: "relative",
+  zIndex: 1010,
+  alignItems: "center",
+  justifyContent: "space-between",
+  display: "flex",
+});
+
+export const Logo = styled("img")({
+  width: 91,
+  height: 34,
+});
+
+export const Nav = styled("nav")({
+  display: "flex",
+  alignItems: "center",
+  maxWidth: 90.34,
+  width: "6.49%",
+  height: 48,
+  color: "#1c1d1f",
+  fontWeight: 400,
+  lineHeight: "1.4",
+  fontSize: "14px",
+  margin: 0,
+  padding: 0,
+  position: "relative",
+});
+
+export const CategoryButton = styled("button")({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  fontWeight: 700,
+  letterSpacing: "-0.32px",
+  fontSize: 14,
+  position: "relative",
+  border: "none",
+  cursor: "pointer",
+  width: "100%",
+  height: 48,
+  color: "#1c1d1f",
+  lineHeight: 0,
+  margin: "12px 0",
+  padding: "0 12px",
+  borderRadius: 0,
+  outline: 0,
+  background: "0 0",
+});
+
+export const Paragraph = styled("span")({
+  fontSize: 14,
+  letterSpacing: "-0.32px",
+  cursor: "pointer",
+  margin: 0,
+  padding: 0,
+  fontWeight: 400,
+  lineHeight: 0,
+  color: "#1c1d1f",
+});
+
+export const HeaderFormField = styled("div")({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  color: "#1c1d1f",
+  fontWeight: 400,
+  lineHeight: 1.4,
+  fontSize: "14px",
+  padding: 0,
+  position: "relative",
+  flexGrow: 1,
+  height: "48px",
+  border: "1px solid #1c1d1f",
+  borderRadius: "9999px",
+  backgroundColor: "#f7f9fa",
+  margin: "0 12px",
+  width: "45.606%",
+});
+
+export const Form = styled("form")({
+  color: "#1c1d1f",
+  fontWeight: 400,
+  lineHeight: 1.4,
+  fontSize: "14px",
+  margin: 0,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  height: "100%",
+  padding: "0 24px 0 0",
+  flexDirection: "row-reverse",
+  width: "98.93208%",
+  position: "relative",
+});
+
+export const FormInputField = styled("input")({
```

Comment:
Shouldn't FormInputField be in a seperate file/common, so that it can be reusused?

I would expect one FormInputField that we can pass props into, to get LoginInput and SearchInput